src/layout/components/NavLeft/Item.vue

```diff
@@ -17,16 +16,29 @@ export default {
     const { icon, title } = context.props
     const vNodes = []
 
+    // 定义文本溢出样式
+    const ellipsisStyle = {
+      overflow: 'hidden',
+      textOverflow: 'ellipsis',
+      whiteSpace: 'nowrap',
+      width: '100%',
+      display: 'inline-block'
+    }
+
     if (icon) {
       if (icon.startsWith('fa-')) {
         vNodes.push(<i class={`fa ${icon}`} />)
       } else {
-        vNodes.push(<svg-icon icon-class={icon}/>)
+        vNodes.push(<svg-icon icon-class={icon} />)
       }
     }
 
     if (title) {
-      vNodes.push(<span slot='title'>{title}</span>)
+      vNodes.push(
+        <el-tooltip content={title} placement='right' effect='dark' open-delay={ 1000 }>
+          <span style={ellipsisStyle}>{title}</span>
+        </el-tooltip>
+      )
     }
     return vNodes
   }
```

Comment:
The code does not contain any irregularities, potential issues, nor is there room for optimization changes.